src/client/entity-editor/common/language-field.tsx

```diff
@@ -61,6 +61,17 @@ function LanguageField({
 	const filterOptions = createFilterOptions({
 		options
 	});
+	const sortFilterOptions = (opts, input, selectOptions) => {
+		const newOptions = filterOptions(opts, input, selectOptions);
+		const sortLang = (a, b) => {
+			if (a.frequency !== b.frequency) {
```

Comment:
I think we should guard against a potential missing frequency property, just in case.
Maybe using the isNumber utility from lodash?
```suggestion
			if (isNumber(a.frequency) && isNumber(B.frequency) && a.frequency !== b.frequency) {
```